sklearn/model_selection/_split.py

```diff
@@ -394,13 +405,13 @@ def get_n_splits(self, X=None, y=None, groups=None):
         Returns
         -------
         n_splits : int
-            Returns the number of splitting iterations in the cross-validator.
+            The number of splitting iterations in the cross-validator.
         """
         return self.n_splits
 
 
 class KFold(_BaseKFold):
-    """K-Folds cross-validator
+    """K-Fold cross-validator.
```

Comment:
Could you include all these little fixes in another PR. I can easily merge it since it will be a doc fix pull-request.